V/VMEC/build_tarballs.jl

```diff
@@ -1,43 +1,106 @@
+  make && make install && make clean
+
+  ./configure CC=gcc FC=gfortran F77=gfortran --with-mkl --host=$target --target=$target --prefix=$prefix
+  # Deal with the issue the gcc doesn't accept -no-undefined at configure step
+  sed "s/AM_LDFLAGS =/AM_LDFLAGS =-no-undefined/" Makefile | cat > Makefile.2
+  mv Makefile.2 Makefile
+  make && make install && make clean
+
+  ./configure CC=gcc FC=gfortran F77=gfortran --host=$target --target=$target --prefix=$prefix
+  # Deal with the issue the gcc doesn't accept -no-undefined at configure step
+  sed "s/AM_LDFLAGS =/AM_LDFLAGS =-no-undefined/" Makefile | cat > Makefile.2
+  mv Makefile.2 Makefile
+  make && make install && make clean
+
+else
+  # Need to apply a fix on apple-darwin* to add missing symlinks for older libmbed* library versions
+  if [[ ${target} == *apple-darwin* ]]; then
+    # Apply libmbed fixes
+    libmbed_path=$(find /workspace/${bb_full_target}/artifacts -name 'libmbedtls.a')
+    artifact_id=$(echo $libmbed_path | grep -o -e "artifacts/[a-z0-9]*" | grep -o -e "/[a-z0-9]*" | grep -o -e "[a-z0-9]*")
+    mbed_version=$(find ${WORKSPACE}/${bb_full_target}/artifacts/${artifact_id}/lib/ -name "libmbedtls.[0-9]*.[0-9]*.[0-9]*.${dlext}" | grep -o -e "[0-9]*\.[0-9]*\.[0-9]*\.[0-9]*")
+    # mbed_version has a leading dot
+    cd ${WORKSPACE}/${bb_full_path}/destdir/lib
+    ln -s ../../artifacts/${artifact_id}/lib/libmbedtls${mbed_version}.${dlext} libmbedtls.13.${dlext}
+    ln -s ../../artifacts/${artifact_id}/lib/libmbedtls${mbed_version}.${dlext} libmbedtls.12.${dlext}
+    ln -s ../../artifacts/${artifact_id}/lib/libmbedcrypto${mbed_version}.${dlext} libmbedcrypto.5.${dlext}
+    ln -s ../../artifacts/${artifact_id}/lib/libmbedcrypto${mbed_version}.${dlext} libmbedcrypto.3.${dlext}
+    ln -s ../../artifacts/${artifact_id}/lib/libmbedcrypto${mbed_version}.${dlext} libmbedcrypto.0.${dlext}
+    ln -s ../../artifacts/${artifact_id}/lib/libmbedx509${mbed_version}.${dlext} libmbedx509.0.${dlext}
+    cd ${WORKSPACE}/srcdir/VMEC*
+  fi
+  # Configure with MKL provided ScaLAPACK
+  ./autogen.sh
+  ./configure CC=mpicc FC=mpifort F77=mpifort FFLAGS=-O3 FCFLAGS=-O3 --with-mkl --enable-tbb --host=$target --target=$target --prefix=$prefix 
+  make && make install && make clean
+  
+  ./configure CC=mpicc FC=mpifort F77=mpifort FFLAGS=-O3 FCFLAGS=-O3 --with-mkl --host=$target --target=$target --prefix=$prefix 
+  make && make install && make clean
+
+  # Configure with OpenBLAS and ScaLAPACK from SCALAPACK_jll
+  ./configure CC=mpicc FC=mpifort F77=mpifort FFLAGS=-O3 FCFLAGS=-O3  --host=$target --target=$target --prefix=$prefix
+  make && make install && make clean
+fi
 """
 
-platforms = [
-    Platform("x86_64", "linux"; libc = "glibc", libgfortran_version="4.0.0"),
-    Platform("x86_64", "linux"; libc = "glibc", libgfortran_version="5.0.0"),
-]
-#platforms = expand_gfortran_versions(platforms)
+incompatible_arch = ["i686", "armv6l", "armv7l", "aarch64", "powerpc64le"]
```

Comment:
Isn't it easier to list the compatible architectures, since it's only one?

Reply:
Yes, that was silly.